main.js

```diff
@@ -57,6 +57,152 @@ buttonElHome.addEventListener('click', () => {
   buttonElHome.classList.add('item__link--position')
 })
 
+// Data
+
+const questionsObj = [
```

Comment:
`questionsObj` is not a good name for an array. Better would be just `questions`.

Reply:
Ok! Will change that. :-)